docs-site/content/guide/high-availability.md

```diff
@@ -272,3 +272,137 @@ final config = Configuration(
 
   </template>
 </Tabs>
+ 
+ ## Deploy a Typesense cluster to Docker Swarm  
+
+To Deploy a Typesense cluster to Docker Swarm, follow these steps:
+
+- Initialize Docker Swarm. For example, in a 4 node setup, target docker engine on one of the nodes to act as a manager
```

Comment:
Did you mean 3 node setup here?

Reply:
4 node - 1 node in Docker Swarm as manager and other 3 nodes as workers to host typesense